### ISSUE TITLE
Nothing for O3DS/2DS?

### ISSUE THREAD

**Issue by paradoX88988** [nedwill/fasthax#11]
Please make something for this!

**Comment by kade-robertson** [comment]
Please don't use the issue tracker to ask for features already being worked on. Just give it time.

**Comment by kurocygnus** [comment]
kade-robertson how do I get the o3ds offsets to try to make it work? I wanna help to develop this.

**Comment by YatoJeimz** [comment]
If nothing comes out for O3DS / 2DS when upgrading to 11.3 nobody can buy a dsiware and downgrade without another DS
If you only focus on N3ds, the O3ds Scene will never progress

Most of those who want the Hax for O3DS are only non-programmers
Only a few of them are programmers and hopefully novice

**Comment by Intege-rs** [comment]
This is a development prototype so other developers can work with it; understand how it works; prove that it works; and find out where it works. Ned said he would be taking a 3 day break before working on this. This is not the final product. I'm pretty sue Ned knows the o3ds offsets or can get them rather easily. He probably didn't include them because he wanted to get this out asap. Without generals knowledge of c all you will be able to do is wait for someone who can finish it finish it. I'm sure there are allot of eager people waiting for this to be finished but your just going to have to wait for Ned or someone else to finish fasthax. (Also it may work on n3ds but it doesn't do anything other than print out "we won" and leave behind some issues that need cleaning up.

Tldr: this is the core of fastback and is missing almost everything useful that a non programmer would use. Even when this is finished we will still have to wait for the homebrew were waiting on to also get updated.

**Comment by YatoJeimz** [comment]
When did He 3 days of rest begin?

**Comment by Intege-rs** [comment]
"till like the 3rd" https://mobile.twitter.com/NedWilliamson/status/813852812543164418 can't copy the tweet directly  because on mobile.

**Comment by kurocygnus** [comment]
Actually, a guy on GBATemp implemented fasthax on FBI and made it to work. (Couldn't test because I have a o3DS)

**Comment by YatoJeimz** [comment]
Could you pass the link for that post? I want to try on my O3DS

**Comment by YatoJeimz** [comment]
What country is ned from?

**Comment by paradoX88988** [comment]
@YatoJemiz  after you've tried give us info if work on O3DS

**Comment by AzizBlooper** [comment]
Last seen Ned on twitter is 30 Dec :v when he want online...I miss him

**Comment by AzizBlooper** [comment]
@YatoJemiz if it work...plz tell us...I want to try to

**Comment by Earned** [comment]
@AzizBlooper Me too :/

**Comment by YatoJeimz** [comment]
@AzizBlooper  I not have the link :"v

**Comment by paradoX88988** [comment]
What link?

**Comment by YatoJeimz** [comment]
The link to the build that says @kurocygnus

**Comment by YatoJeimz** [comment]
But I already used the alpha build that send Ned and it froze until the 3 attempts the exploit was executed and it gave an error of K11

**Comment by paradoX88988** [comment]
Go on gbatemp forum

**Comment by AzizBlooper** [comment]
Sorry but gbatemp not help me very much

**Comment by YatoJeimz** [comment]
Search all over GBATEMP and found nothing on a build with FBI support

**Comment by Irastris** [comment]
Oh please, that guy didn't "implement" anything. I guarantee you he just used Mrrraou's fork of FBI, which searches for an existing k11 exploit instead of attempting waithax.
